lightyear/src/client/prediction/rollback.rs

```diff
@@ -932,6 +943,109 @@ mod integration_tests {
         (stepper, confirmed, predicted)
     }
 
+    /// Test that the entities of a predicted component marked as to be
+    /// entity-mapped are mapped when rollbacked.
+    #[test]
+    fn test_rollback_entity_mapping() {
+        #[derive(Component, Serialize, Deserialize, Clone, Copy, PartialEq)]
+        struct ComponentWithEntity(Entity);
```

Comment:
I already provide a `ComponentMapEntities` in shared/tests/protocol if you want to skip this setup code

Reply:
`ComponentMapEntities` uses simple prediction. Trying to override its prediction mode to full doesn't seem to work.